app/controllers/admin/community_customizations_controller.rb

```diff
@@ -23,16 +28,24 @@ def update_details
         :transaction_agreement_content
       ]
       params.require(:community_customizations).require(locale).permit(*permitted_params)
-
       locale_params = params[:community_customizations][locale]
       customizations = find_or_initialize_customizations_for_locale(locale)
       customizations.update_attributes(locale_params)
     end
 
+    enabled_locales = params[:enabled_locales]
+    all_locales = MarketplaceService::API::Marketplaces.all_locales
```

Comment:
Minor perf improvement: With `to_set` you can convert array to Set (http://ruby-doc.org/stdlib-2.2.2/libdoc/set/rdoc/Set.html). Set allows faster constant time look up, which improves the perf of the loop in loop code.